src/theme/dojo/loading-indicator.m.css

```diff
@@ -7,6 +10,15 @@
 	width: 100%;
 }
 
+.circular {
+	border: calc(var(--grid-base) / 2) solid var(--color-background-faded);
+	border-radius: 50%;
+	border-top: calc(var(--grid-base) / 2) solid var(--color-background-inverted);
+	width: calc(var(--grid-base) * 6.25);
```

Comment:
I'm not sure that these sizes here etc are relevant to the grid size, ie. if the grid size changed would you expect the loader to change size?
Also, we should really refrain from fractional multipliers like these. Just hardcode the width / height appropriately  in this case.

Reply:
Removed the width/height here as they're not needed